### PR TITLE
Pod affinity namespaceSelector to beta

### PR DESCRIPTION
**Pull request by ahg-g**
ref: kubernetes/enhancements#2249

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ different Kubernetes components.
 | `NonPreemptingPriority` | `false` | Alpha | 1.15 | 1.18 |
 | `NonPreemptingPriority` | `true` | Beta | 1.19 | |
 | `PodDeletionCost` | `false` | Alpha | 1.21 | |
-| `PodAffinityNamespaceSelector` | `false` | Alpha | 1.21 | |
+| `PodAffinityNamespaceSelector` | `true` | Beta | 1.21 | 1.22 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:150]:
```suggestion
| `PodAffinityNamespaceSelector` | `false` | Alpha | 1.21 | 1.21 |
| `PodAffinityNamespaceSelector` | `true` | Beta | 1.22 |  |
```

**Review by sftim — CHANGES_REQUESTED** [review]
I've added a suggestion to show how to update the feature gate table.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (ahg-g) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: done.

**Pull request merged**